docs/Upgrade-Guide/upgrade_to_8.md

```diff
@@ -0,0 +1,42 @@
+# Upgrade procedure to Gluster 8, from Gluster 7.x, 6.x, 5.x, 4.1.x, and 3.12.x
+
+We recommend reading the [release notes for 8.0](../release-notes/8.0.md) to be
+aware of the features and fixes provided with the release.
+
+> **NOTE:** Upgrade procedure remains the same as with 4.1.x release
+
+Refer, to the [Upgrading to 4.1](./upgrade_to_4.1.md) guide and follow
+documented instructions, replacing 8 when you encounter 4.1 in the guide as the
+version reference.
+
+### Major issues
+
+1. The following options are removed from the code base and require to be unset
```

Comment:
It is good to mention the other features deprecated in the older version and this version as well. Like tier, stripe and so on. This wasn't done in the past, but its a good thing to correct. As these changes can also break the system.